backend/config/settings.py

```diff
@@ -284,6 +286,9 @@
     # One of the Cloud.gov environments
     STATICFILES_STORAGE = "storages.backends.s3boto3.S3ManifestStaticStorage"
     DEFAULT_FILE_STORAGE = "report_submission.storages.S3PrivateStorage"
+
+    delete_env_var("DATABASE_URL")
```

Comment:
Per chat, when you change an env var using Python, those changes only apply to the process (and child processes) that made the changes in the first place. This means the variable will persist at the OS level, despite being unset as far as the running Django process is concerned.

For our purposes, deleting `DATABASE_URL` for Django's running process may be sufficient since, as far as we can tell, nothing else is referencing it.